detox/src/devices/runtime/drivers/android/AndroidDriver.js

```diff
@@ -394,6 +410,43 @@ class AndroidDriver extends DeviceDriverBase {
       'Press any key to continue...'
     );
   }
+
+  _getLocalBinaryHash(binaryFile) {
+    return generateHash(binaryFile);
+  }
+
+  async _saveHashToRemote(hash, bundleId) {
+    await this.hashHelper.saveHashToRemote(this.adbName, bundleId, hash);
```

Comment:
this should get the hash file name as a parameter (and not to generate it by itself)